### PR TITLE
Update all usage of Tumblr's CSS variables

### PR DESCRIPTION
**Pull request by AprilSylph**
most notably fixes AlreadyReblogged, Tag Tracking+ being readable on dark mode, and Reblog Timestamps being the right colour

let me know if it would be better to split this up per extension

### CODE REVIEW

**Review by hobinjk — APPROVED** [review]
Don't have the time to test locally but code LGTM

File: Extensions/blacklist.js [hunk 1]
```diff
@@ -179,7 +179,7 @@ XKit.extensions.blacklist = new Object({
 				.xblacklist_blacklisted_post .xblacklist_excuse {
 					height: 40px !important;
 					line-height: 40px !important;
-					color: var(--transparent-white-40, rgba(255,255,255,.43));
+					color: rgba(var(--white-on-dark, 255,255,255), 0.4);
```
[the remark this answers is not in the document]
Comment (AprilSylph) [reply Extensions/blacklist.js:182]: I was worried that it wouldn't work, but 1. `var()` only allows one fallback value, and 2. MDN specifies that the only illegal characters in the fallback value are "newlines, unmatched closing brackets, i.e. `)`, `]`, or `}`, top-level semicolons, or exclamation marks". I'll admit that Blacklist is the only thing here that I didn't test though, so one of us probably should do that before merging

File: Extensions/tweaks.js [hunk 2]
```diff
@@ -807,7 +807,7 @@ XKit.extensions.tweaks = new Object({
 		if (XKit.extensions.tweaks.preferences.hide_activity_notification_badge.value && XKit.page.react) {
 			let activityAriaLabel = await XKit.interface.translate('Activity');
 			let notificationBadgeSel = XKit.css_map.keyToClasses('notificationBadge').map(cssClass => `button[aria-label="${activityAriaLabel}"] .${cssClass}`).join(',');
-			XKit.extensions.tweaks.add_css(`${notificationBadgeSel} { 
+			XKit.extensions.tweaks.add_css(`${notificationBadgeSel} {
```
[the remark this answers is not in the document]
Comment (AprilSylph) [reply Extensions/tweaks.js:810]: genuinely didn't even notice this. yay for atom+editorconfig

**Comment by AprilSylph** [comment]
i'm gonna hold off on merging this until monday morning because ~~i don't want to potentially mess up my neat contribution graph~~ if any issues arise from this i'd prefer to be able to jump right on them

**Pull request merged**